dev/Deployment/DeploymentManager.cpp

```diff
@@ -274,7 +295,14 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
             packagePath /= package.identifier + WINDOWSAPPRUNTIME_FRAMEWORK_PACKAGE_FILE_EXTENSION;
 
             // Deploy package.
-            RETURN_IF_FAILED(AddPackage(packagePath));
+            RETURN_IF_FAILED(AddPackage(packagePath, forceDeployment));
+        }
+
+        // Restart Push Notifications Long Running Platform when ForceDeployment option is applied.
+        if (forceDeployment)
+        {
+            wil::com_ptr_nothrow<INotificationsLongRunningPlatform> longRunningProcessPlatform{
```

Comment:
We talked about a 3 time retry policy here as well if it fails

Reply:
Yes, that is still the plan for the next commit update for the PR. First, I am looking to verify LRP restart change is working. Installer exe is not part of WAR dll and is not built in this repo's build pipelines but in internal pipelines due to dependencies. So, to test it from Installer, I have to run the internal pipeline to refer to this private branch.